### DISCUSSION TITLE
[Feature] "smart" albums

### DISCUSSION THREAD

**Discussion by droopified** [immich-app/immich#11228] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

i want something similar to Plex's "smart" playlist where if i create an album and tell it to include faces: x, y, x persons, every time a new picture is added to the library and a face is added to that image, auto add it to the album because it contains x person. 

but not just faces, maybe a combination of things or keywords. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by pelaxa** [comment]
how come this was closed with no comments?  I am also looking for this feature

**Reply by bo0tzz** [reply]
#1673
